.github/workflows/nix-full.yml

```
@@ -0,0 +1,47 @@
+name: Nix Full Validation
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["CI"]
+    types: [completed]
+    branches: [ "main" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)